examples/react-custom-fonts/src/App.jsx

```diff
@@ -0,0 +1,51 @@
+import './App.css'
+
+import {Cloudinary} from "@cloudinary/url-gen"
+import {source} from "@cloudinary/url-gen/actions/overlay"
+import {scale} from "@cloudinary/url-gen/actions/resize"
+import {text} from "@cloudinary/url-gen/qualifiers/source"
+import {compass} from "@cloudinary/url-gen/qualifiers/gravity";
+import {Position} from "@cloudinary/url-gen/qualifiers/position";
+import {TextStyle} from "@cloudinary/url-gen/qualifiers/textStyle"
+
+
+const cloudinary = new Cloudinary({
+    cloud: {
+        cloudName: import.meta.env.VITE_CLOUDINARY_CLOUD_NAME,
+    },
+})
+
+const cldImage = cloudinary
+    .image("examples/sneakers_cuuqb1.jpg")
```

Comment:
can you update this public ID to `cld-sample-5`? im going to start encouraging use of images from the `samples` directory or samples that exist in all new accounts ot make it easier for people to spin these projects up